### PR TITLE
upgrade unsafe

### PR DESCRIPTION
**Pull request by Devan-Kerman**

### CODE REVIEW

**Comment by Devan-Kerman** [comment]
nou

**Comment by Devan-Kerman** [comment]
noae

**Comment by Devan-Kerman** [comment]
uar usnafe is atilel thair

**Comment by Devan-Kerman** [comment]
i jus upgardeted t

File: src/main/java/io/github/fabriccommunity/everything/ExampleMod.java [hunk 1]
```diff
@@ -21,11 +21,6 @@ public void onInitialize() {
 		a51cd4448b2346938cf395dcb2cf3229.c441f7b88bb04e7a9d4d13b75703fcea();
 
 		System.out.println("Hello Fabric world!");
-		try {
```
Comment (valoeghese) [inline src/main/java/io/github/fabriccommunity/everything/ExampleMod.java:24]: why

Comment (vini2003) [reply src/main/java/io/github/fabriccommunity/everything/ExampleMod.java:24]: idk but i like it

**Comment by vini2003** [comment]
i request changes

File: src/main/java/io/github/fabriccommunity/everything/ExampleMod.java (hunk 1 above)
Comment (Devan-Kerman) [reply src/main/java/io/github/fabriccommunity/everything/ExampleMod.java:24]: yes

**Pull request merged**